### ISSUE TITLE
Installation language should be asked by installation iso

### ISSUE THREAD

**Issue by NOhs** [GPMueller/arch-installer#74]
Formatting the disk with the wrong keyboard layout is a pain and unacceptable. We should consider asking for the language immediately.

**Comment by GPMueller** [comment]
I agree that this makes sense for the live iso, so we should add a script (bash?) to the iso.
Should the installer still ask for language settings or should this be inferred from the iso?
